### DISCUSSION TITLE
about 无gt推理

### DISCUSSION THREAD

**Discussion by frensher** [open-mmlab/mmdetection#8894] [Q&A]
你好,我想问下,我按照这个操作,进行无gt的推理.
输出json,却报错了.
我在images2coco.py 中设置的class.txt就是coco的八十个类.
其他无改动
![图片](https://user-images.githubusercontent.com/42425485/187885146-17ef6ca1-6624-42ad-819f-4c5fa946c64f.png)
https://github.com/open-mmlab/mmdetection/blob/ca11860f4f/docs/zh_cn/1_exist_data_model.md#%E4%B8%8D%E4%BD%BF%E7%94%A8-ground-truth-%E6%A0%87%E6%B3%A8%E8%BF%9B%E8%A1%8C%E6%B5%8B%E8%AF%95
![图片](https://user-images.githubusercontent.com/42425485/187885634-84d3db73-c9cf-491f-8263-a8b69d7a5f9c.png)
是因为图片完全没有检测出来任何东西吗

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by frensher** [comment]
> 你好,我想问下,我按照这个操作,进行无gt的推理. 输出json,却报错了. 我在images2coco.py 中设置的class.txt就是coco的八十个类. 其他无改动 ![图片](https://user-images.githubusercontent.com/42425485/187885146-17ef6ca1-6624-42ad-819f-4c5fa946c64f.png) https://github.com/open-mmlab/mmdetection/blob/ca11860f4f/docs/zh_cn/1_exist_data_model.md#%E4%B8%8D%E4%BD%BF%E7%94%A8-ground-truth-%E6%A0%87%E6%B3%A8%E8%BF%9B%E8%A1%8C%E6%B5%8B%E8%AF%95 ![图片](https://user-images.githubusercontent.com/42425485/187885634-84d3db73-c9cf-491f-8263-a8b69d7a5f9c.png) 是因为图片完全没有检测出来任何东西吗

我换了其他的图片集也不行啊

**Comment by ZwwWayne** [comment]
This usually means that the number of classes in the model does not match that of the dataset.
